rohmu/object_storage/google.py

```diff
@@ -426,17 +452,17 @@ def get_file_size(self, key):
 
     def _upload(
         self,
-        upload,
-        key,
-        metadata,
-        extra_props,
-        cache_control,
+        upload: MediaUpload,
+        key: str,
+        metadata: Metadata,
+        extra_props: Optional[Dict[str, Any]],
+        cache_control: Optional[str],
         reporter: Reporter,
         upload_progress_fn: IncrementalProgressCallbackType = None,
-    ):
+    ) -> Any:
```

Comment:
No need for this return value - I think google implementations store_file_object returning something is a bug, not a feature (others do not return anything and in general API should be what is in base.py).